test/configCases/css/css-module-scripts-template-literal/index.js

```diff
@@ -0,0 +1,14 @@
+import sheet from './style.css' assert { type: 'css' }
+import basicSheet from './basic.css' assert { type: "css"}
```

Comment:
Let add here `import basicSheet from './basic.css'` without `assert { type: "css"}`  to ensure we are processing `import`'s the same things twice correctly

Reply:
The expected result of import basicSheet1 from './basic.css' is that a warning should be emitted about the module default export not found, basicSheet1 should be undefined, and bundle0.css should be created correctly.